### ISSUE TITLE
Will the pruned weight reactivated after finetuning?

### ISSUE THREAD

**Issue by igo312** [jacobgil/pytorch-pruning#41]
@jacobgil I find there is no limits when using `optimizer.step()`. So the pruned weight will get a gradient and after stepping, it will be no longer **0** which means it cannot be regarded as pruned? 

Am I right? Hope for your response!
